### ISSUE TITLE
Detect drift of Seed backed job-type

### ISSUE THREAD

**Issue by gisjedi** [ngageoint/scale-ui#388]
When viewing an existing job type, the UI should make a request to Silo to identify any drift of configuration from the Seed specified manifest. Scale can have a job type configuration mismatch both intentionally (resources overridden) and unintentionally (up-to-dated interface in Scale, but failure to apply back to code-base / Seed Docker label). We should at minimum flag that when you open up a job type. Best case would be to show a side-by-side comparison of the current job type manifest and the Seed specified manifest to quickly identify the drift.

**Comment by cshamis** [comment]
I think we need to consider this function with a signing function at the
same time.  Drift or tampering, the difference is the same, the intention
is not.  But we don’t need to consider intention if we solve the problem.

**Comment by gisjedi** [comment]
You are correct, though my take was that unintentional changes were due to lack of awareness as opposed to nefarious intent. I think the solution is the same. We classify 2 types of change: manifest and image. 

A manifest change indicates we have manipulated some portion of the manifest JSON within scale so it doesn't perfectly track with the one embedded with the image.

An image change indicates that the hash of the job image itself doesn't match with what we imported. This can be either an intentional change - such as pushing over the same image tag. Or it could be an unintentional one - a malicious third party has updated the image.

This conversation also tends to bleed into one on algorithm signing. That is a much more challenging discussion and one that should be addressed later. This is just to guarantee integrity, signed approvals are a much bigger issue.
